packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -78,6 +112,15 @@ export const prepareLocalchainAccountKit = (baggage, makeRecorderKit, zcf) => {
         getUpdater() {
           return this.state.topicKit.recorder;
         },
+        /**
+         * @type {ReturnType<typeof makeTimestampHelper>['getCurrentTimestamp']}
+         */
+        async getTimeoutTimestamp(relativeTime) {
```

Comment:
once `timer` and `timerBrand` are pulled up, this needn't be on a facet. it can be in the closure too.

```js
  const timestampHelper = makeTimestampHelper(timer, timerBrand);
```

Reply:
Thanks, artifact as this wasn't initially factored out. _Planning to reuse it for ICA transfers_